framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -703,13 +704,15 @@ private void standardizeAtm(
      * @param context the context
      * @param localScope the local scope
      * @param useLocalScope whether {@code localScope} should be used to resolve identifiers
+     * @param atMethodSignature if true, use "#2" instead of names for formal parameters
      * @return the standardized version of the Java expression
      */
     protected String standardizeString(
             String expression,
             JavaExpressionContext context,
             TreePath localScope,
-            boolean useLocalScope) {
+            boolean useLocalScope,
+            boolean atMethodSignature) {
```

Comment:
It's weird that this parameter only need for the subclasses.  I think this pull request breaks the code that makes it easy to add a contract annotation of a type system with dependent types using `org.checkerframework.framework.qual.QualifierArgument`.  See test case, `checker/tests/nullness/EnsuresKeyForOverriding.java` that passes on master, but fails on this branch.

Reply:
Thanks for the test case!  I will take a look.

Reply:
See https://github.com/mernst/checker-framework/pull/11 which removes this.